src/ArrayValueObject/ArrayValueObject.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ValueObjects\ArrayValueObject;
+
+use ValueObjects\AbstractValueObject;
+use ValueObjects\Exception\ArrayValueObject\ArrayValueObjectInvalidException;
+
+class ArrayValueObject extends AbstractValueObject
```

Comment:
Hi Andrés,

Not sure we need an Array ValueObject. We can just use a Collection of VOs that accomplish the same objective.

Reply:
Hi Manel,

Thanks for your response. You're right, I Just saw that the collection VO only accepts one type of VO and maybe in some cases you just want to have an Array VO that can be extended to add extra validations, for example: an array of ids that can contain different types of ids, extending from this ArrayValueObject you can create an IdsArrayValueObject that have some extra validation for each ID other way you need to create different collections to validate each collection of IDs individually.

What do you think?